config/samples/networking_v1alpha_location.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: networking.datumapis.com/v1alpha
+kind: Location
+metadata:
+  name: us-central1-a
+  namespace: default
+spec:
+  # The location class that indicates control plane behavior
+  # Valid values are: datum-managed, self-managed
+  locationClassName: datum-managed
+  
+  # The topology of the location
+  topology:
+    topology.datum.net/city-code: "us-central1"
```

Should use a city code that'll work for IPAM today (Only DFW, DLS, and LHR will work at the moment):

```suggestion
    topology.datum.net/city-code: "DFW"
```